themes/src/main/resources-community/theme/base/account/messages/messages_de.properties

```diff
@@ -352,4 +352,4 @@ openshift.scope.user_check-access=Benutzerzugriffsinformationen
 openshift.scope.user_full=Voller Zugriff
 openshift.scope.list-projects=Projekte auflisten
 
-locale_fa=\u0641\u0627\u0631\u0633\u06CC
\ No newline at end of file
+locale_fa=\u0641\u0627\u0631\u0633\u06CC
```

Comment:
@jonkoops I was once taught that text files should generally end with a line ending because it makes handling them easier e.g. for concatenation. So I assumed this would be a valid fix to the missing line ending...

Reply:
It doesn't really matter, this is a nit either way. The concatenation argument is a tad outdated I think, I would not see a situation where we would do that, and even if we did newlines should be handled properly.

Either way, I don't care enough to hold this up or anything :stuck_out_tongue: